.github/workflows/full_stack_tests.yml

```
@@ -318,11 +318,11 @@ jobs:
           name: generated-test-to-angular-module-mapping-${{steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME}}
           path: /home/runner/work/oppia/oppia/core/tests/test-modules-mappings/acceptance/${{ steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME }}.txt
 
-      - name: Upload webpack bundles
+      - name: Upload build artifacts
         if: ${{ failure() && (steps.desktop-test-attempt-1.outcome == 'failure' || steps.mobile-test-attempt-1.outcome == 'failure') }}
```

actionlint (1.7.12)

[error] property "desktop-test-attempt-1" is not defined in object type {check_skip: {conclusion: string; outcome: string; outputs: {string => string}}; desktop-test: {conclusion: string; outcome: string; outputs: {}}; generate_suite_name_for_artifacts: {conclusion: string; outcome: string; outputs: {string => string}}; mobile-test: {conclusion: string; outcome: string; outputs: {}}}

(expression)